src/PluginInfo/PluginInfo.js

```diff
@@ -130,12 +131,13 @@ function PluginInfo (dirname) {
 
     function _parseConfigFile (tag) {
         var configFile =
-            { target: tag.attrib['target'],
-                parent: tag.attrib['parent'],
-                after: tag.attrib['after'],
+            {
```

Comment:
Can we bring this up to the assignment line and unindent the object body please?